### PR TITLE
pip-install: add German translation

### PR DESCRIPTION
**Pull request by dadav**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages.de/common/pip-install.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+> Installiere Python-Pakete.
+> Weitere Informationen: <https://pip.pypa.io>.
+
+- Installiere ein Paket:
+
+`pip install {{paketname}}`
+
+- Installiere eine spezifische Paketversion:
+
+`pip install {{paketname}}=={{paketversion}}`
```
Comment (navarroaxel) [inline pages.de/common/pip-install.md:12]:
```suggestion
`pip install {{paket_name}}=={{paket_version}}`
```

File: pages.de/common/pip-install.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# pip install
+
+> Installiere Python-Pakete.
+> Weitere Informationen: <https://pip.pypa.io>.
+
+- Installiere ein Paket:
+
+`pip install {{paketname}}`
```
Comment (navarroaxel) [inline pages.de/common/pip-install.md:8]:
```suggestion
`pip install {{paket_name}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.de/common/pip-install.md (hunk 1 above)
Comment (dadav) [reply pages.de/common/pip-install.md:12]: hm. In german this is one word, that's why I chose to make it one word here. Do you still want me to split it?

Comment (navarroaxel) [reply pages.de/common/pip-install.md:12]: Ok, I don't speak German. @marchersimon, could you help?

Comment (marchersimon) [reply pages.de/common/pip-install.md:12]: Yep, it's indeed just one word, so we sholud leave it like it is.

File: pages.de/common/pip-install.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Installiere eine spezifische Paketversion:
+
+`pip install {{paketname}}=={{paketversion}}`
+
+- Installiere die Pakete aus einer Datei:
+
+`pip install -r {{requirements.txt}}`
+
+- Installiere die Pakete von einer URL oder eines lokalen Archivs (.tar.gz | .whl):
```
Comment (marchersimon) [inline pages.de/common/pip-install.md:18]:
```suggestion
- Installiere die Pakete von einer URL oder einem lokalen Archiv (.tar.gz | .whl):
```

File: pages.de/common/pip-install.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`pip install -f {{url|pfad/zur/datei}}`
+
+- Installiere das Paket den Quellen des aktuellen Verzeichnisses:
```
Comment (marchersimon) [inline pages.de/common/pip-install.md:22]: Why did you reformulate the sentence? I can't tell if it's still correct and I can't really make sense of it. How about
```suggestion
- Installiere das lokale Paket im aktuellen Verzeichnis im Entwichlungs-/Bearbeitungsmodus:
```

**Review by marchersimon — APPROVED** [review]
Thanks @dadav

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**